src/main/java/com/eva/logic/commands/CommentCommand.java

```diff
@@ -29,8 +29,9 @@
 
     public static final String COMMAND_WORD = "comment";
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds or delete comment to person"
-            + "To add: key in '-a t: {Title} d: {Date} desc: {Description}'"
-            + "and to delete, key in '-d t:{TitletoDelete}'";
+            + "To add comment: key in 'addComment INDEX <s-/a-> i/TITLE d/DATE desc/DESCRIPTION'\n"
```

Comment:
Possible to standardise the prefixes for the title when adding or deleting the comment?